src/features/routes/routes.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Link, Switch, Route, useParams, useRouteMatch, useHistory } from 'react-router-dom';
+
+import { Counter } from '../index.js';
+import { Button } from '../../components/index.js'
+
+export const Routes = () => (
+    <>
```

Comment:
could u please divide routing and markup?

Reply:
It's just an example of usage. Later, only Switch case with Route will remain.